### DISCUSSION TITLE
How to change the width and height of an image in batch generation ?

### DISCUSSION THREAD

**Discussion by FunnyFinger** [Comfy-Org/ComfyUI#5598] [Q&A] [answered]
I want to simply generate two images with the same aspect ratio, but one in portrait and one in landscape using the same setting of everything else. I want to do even more like generating a batch of images with width and height are from a list  of numbers, like for example : width={200, 500 , 600} height={600, 500, 200}. it's like XY plot but here I'm only changing the width and height.

**Comment by ltdrdata** [comment]
Convert the width and height to 'Convert to Input', then use a text manipulation node to generate a list output, and use a node that converts text to integer to input values into width and height respectively.

Take a look at this:
https://github.com/Suzie1/ComfyUI_Comfyroll_CustomNodes
https://github.com/WASasquatch/was-node-suite-comfyui

**Reply by FunnyFinger** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/1f6ff811-49a8-4818-b6a2-9da8cea6766d)

I found a way to do it using CR Value Cycler Node, I'm keeping the batch size to 1. it generate 2 images,  1280x720 & 720x1280.
